src/jvm/main/org/jetbrains/kotlinx/lincheck/trace/TracePoint.kt

```diff
@@ -538,6 +522,14 @@ internal class SpinCycleStartTracePoint(iThread: Int, actorId: Int, callStackTra
     }
 }
 
+internal class ReturnFromMethodTracePoint(iThread: Int, actorId: Int, callStackTrace: CallStackTrace, val methodName: String): TracePoint(iThread, actorId, callStackTrace) {
```

Comment:
Let's rename into one of the following:
- `MethodResultTracePoint`
- `MethodReturnTracePoint`
- `MethodExitTracePoint`